core/contracts/common/test/TestableTest.sol

```diff
@@ -6,7 +6,7 @@ import "../implementation/Testable.sol";
 // TestableTest is derived from the abstract contract Testable for testing purposes.
 contract TestableTest is Testable {
     // solhint-disable-next-line no-empty-blocks
-    constructor(bool _isTest) public Testable(_isTest) {}
+    constructor(bool _isTest, address _timerAddress) public Testable(_isTest, _timerAddress) {}
```

Comment:
Having both `isTest` and `_timerAddress` feels redundant to me. Could you not enfer the value of isTest from `timerAddress`? So if `timerAddress == address(0x0)` then you know you in mode `isTest=False`?

Reply:
That's a good point, cuts down on number of input variables as well!

Reply:
Yup. It'll be the same as before :)

Reply:
I made this change